lib/Member.php

```diff
@@ -78,7 +77,7 @@ public function getLastHash()
      *
      * @return RepeatedField a list of aliases
      */
-    public function getAliases()
+    public function aliases()
```

Comment:
We need to align `aliases` and `getFirstAlias`, i.e. either both have `get` or both not. I actually prefer `get`. I have we used to cache aliases in the SDK, which explains why in Java SDK this call is just `aliases`, but since now we are actually making API calls, I prefer `getAliases`. @gianluca-token Opinion?